src/components/views/elements/Field.tsx

```diff
@@ -85,6 +89,8 @@ export interface IInputProps extends IProps, InputHTMLAttributes<HTMLInputElemen
 interface ISelectProps extends IProps, SelectHTMLAttributes<HTMLSelectElement> {
     // The ref pass through to the select
     inputRef?: RefObject<HTMLSelectElement>;
+    // Ref callback that will be attached to the input. This takes precedence over inputRef.
+    refCallback?: RefCallback<HTMLSelectElement>;
```

Comment:
Seems that `refCallback` is incompatible with the `focus` method on `Field` - why two different fields rather than just accepting any Ref if that is the intent?

Reply:
Have made it better in https://github.com/element-hq/element-web/pull/28452/commits/e5c0cdc402d7aa5c708811904f1bd91b1c8eeefa